torch/utils/optimizer.py

```diff
@@ -0,0 +1,26 @@
+"""
```

Comment:
Should we name this file `mobile_optimizer`? True that our prepacking and packed op optimizations are not strictly mobile specific, but I wonder if we will want to put other non-mobile specific optimization in `torch.utils.optimizer` namespace.
If we do, maybe we should get a consensus from others who will want to have their optimizations places here? Maybe quantization team? @AshkanAliabadi @dreiss your opinions?

Reply:
If we move it to mobile folder, does it still necessary to check the name into mobile_optimizer?